public/pages/utils/constants.ts

```diff
@@ -16,13 +16,18 @@
 import { SORT_DIRECTION } from '../../../server/utils/constants';
 import { DETECTOR_STATE } from '../../utils/constants';
 
+export const newSubduedColor = '#98A2B3';
```

Comment:
`CustomizedSubduedColor` may be a better name. Same for below new colors.

Reply:
Agreed, will change